jpa/src/main/java/org/jboss/as/jpa/container/ExtendedEntityManager.java

```diff
@@ -89,8 +89,6 @@
      */
     private final ExtendedEntityManagerKey ID = ExtendedEntityManagerKey.extendedEntityManagerID();
 
-    private final transient boolean isTraceEnabled = ROOT_LOGGER.isTraceEnabled();
```

Comment:
The scope of the object here is more likely to be per referencing stateful bean(s).  I haven't gotten any complaints about this.  If TRACE logging is enabled after starting the app server (if that is possible?), the application would have to either get lucky (create new stateful beans) or redeploy to cause new instances to be created.